setup.cfg

```diff
@@ -0,0 +1,58 @@
+[metadata]
+
+name = projectq
+version = file:VERSION.txt
+url = http://www.projectq.ch
+author = ProjectQ
+author_email = info@projectq.ch
+project_urls =
+       Documentation = https://projectq.readthedocs.io/en/latest/
+        Issue Tracker = https://github.com/ProjectQ-Framework/ProjectQ/
+license = Apache License Version 2.0
+license_file = LICENSE
+description = ProjectQ - An open source software framework for quantum computing
+long_description = file:README.rst
+long_description_content_type = text/x-rst; charset=UTF-8
+home_page = http://www.projectq.ch
+requires_dist = setuptools
+classifier =
+    License :: OSI Approved :: Apache Software License
+    Topic :: Software Development :: Libraries :: Python Modules
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+
+[options]
+
+zip_safe = False
+packages = find:
+python_requires = >= 3
+install_requires =
+    matplotlib >= 2.2.3
+    networkx >= 2
+    numpy
+    requests
+    scipy
```

Comment:
What about pybind11?

Reply:
Not required for running ProjectQ only for the building part.

Reply:
Right, ok.